web/src/components/topology/Korrel8rTopology.tsx

```diff
@@ -26,17 +26,15 @@ import {
 } from '@patternfly/react-topology';
 import * as React from 'react';
 import { TFunction, useTranslation } from 'react-i18next';
-import { useHistory, useLocation } from 'react-router';
-import { nodeToLabel } from '../../korrel8r-utils';
-import { InvalidNode } from '../../korrel8r/invalid';
-import { Korrel8rNode } from '../../korrel8r/korrel8r.types';
-import { Korrel8rNodeFactory } from '../../korrel8r/node-factory';
-import { Edge as EdgeData } from '../../korrel8r/client/models/Edge';
-import { Node as NodeData } from '../../korrel8r/client/models/Node'; // FIXME use as Data
-import { Query, QueryType } from '../../redux-actions';
+import { useHistory } from 'react-router';
```

Comment:
The console is deprecating the react-router v5 package, so we are swapping to the compat package (which basically uses v6 under the hood) provided by the console. In react-router v6, the `useHistory` hook is replaced with the `useNavigate` hook. 

https://reactrouter.com/6.30.0/upgrading/v5#use-usenavigate-instead-of-usehistory

Jenny has already made this change and it has been merged into main, so a rebase should be able to fix it.

```suggestion
import { useNaviate } from 'react-router-dom-v5-compat';
```